astrokat/__init__.py

```diff
@@ -1,7 +1,11 @@
+from __future__ import division
+from __future__ import absolute_import
+
 from .__main__ import cli  # noqa
-import noisediode  # noqa
-import correlator  # noqa
-import scans # noqa
+
+from .noisediode  import * # noqa
+from .correlator  import * # noqa
+from .scans import * # noqa
```

Comment:
This doesn't look the same as it used to be.  Previously we imported the modules not everything inside them - this will change the global namespace, or at least pollute it.  E.g. If we've done `import astrokat`, then we can access the `noisediode`'s `on` function as `astrokat.on`.

Rather:
```python
from . import noisediode
from . import correlator
from . import scans
```